deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/FileUploadCompletionNotification.java

```diff
@@ -66,7 +66,7 @@ public FileUploadCompletionNotification(String correlationId, Boolean isSuccess,
      * Construct this notification with json
      * @param json the json to parse.
      */
-    public FileUploadCompletionNotification(String json)
+    private FileUploadCompletionNotification(String json)
```

Comment:
This seems like the right thing to do. 

Would we ever want a customer to construct their own notification this way? Seems like we wouldn't.

Reply:
No, this is a read-only field that we return. Users never create one to provide to our SDK